DIPs/DIP-3.md

```diff
@@ -0,0 +1,54 @@
+---
+Title: Flight and hotel bookings sourced from decentralized open-source travel marketplace
+Primary Contact: <Maksim Izmaylov> max@windingtree.com
+Themes: Ticketing, Purchases & ID
+Tags: Attendee Experience
+Resources Required: Some Devcon dev time, connecting about hotels in Bogota
+Discussions: https://forum.devcon.org/t/flights-and-hotels-sourced-via-decentralized-marketplace/42
+---
+
+## Summary
+
+Winding Tree offers Devcon attendees to book their flights and hotels directly from suppliers with discount. Winding Tree is 100% open-source and we aim to be as decentralized as it is possible.
+
+## Abstract
+
+Winding Tree is Ethereum-based decentralized B2B travel marketplace. We've also built a set of open source tools, including a [travel agency](https://glider.travel), which we could integrate directly into Devcon website. We'll also take on the responsibility of finding discounted hotel rates in Bogota, and also discounted flight tickets.
+
+## Motivation & Rationale
+
+It would be grear if Devcon attendees were able to book their flight and hotel immediately after purchasing their Devcon ticket, and 10-20% cheaper than on any other travel booking website!
```

Comment:
```suggestion
It would be great if Devcon attendees were able to book their flight and hotel immediately after purchasing their Devcon ticket, and 10-20% cheaper than on any other travel booking website!
```